.rubocop.yml

```diff
@@ -0,0 +1,19 @@
+Documentation:
+    Enabled: false
+Metrics/LineLength:
+  Max: 100
+AllCops:
+  Exclude:
+    - 'db/**/*'
+    - 'vendor/**/*' 
+    - 'config/**/*'
+    - 'bin/**/*'
+    - 'test/test_helper.rb'
+    - 'app/models/application_record.rb'
+    - 'app/channels/**/*'
+    - 'app/helpers/**/*'
+    - 'app/controllers/application_controller.rb'
```

Comment:
Basically there is better to not exclude `app/models/application_record.rb` and `app/controllers/application_controller.rb` files but in our case this is OK because they're empty